Formula/micro.rb

```diff
@@ -16,15 +16,10 @@ class Micro < Formula
 
   def install
     ENV["GOPATH"] = buildpath
-    mkdir_p buildpath/"src/github.com/zyedidia/micro"
 
-    dir = buildpath/"src/github.com/zyedidia/micro"
-    dir.install (buildpath/"cmd")
-    dir.install (buildpath/"tools")
-    dir.install (buildpath/"runtime")
-    dir.install (buildpath/"Makefile")
+    (buildpath/"src/github.com/zyedidia/micro").install buildpath.children
 
-    cd dir do
+    cd "src/github.com/zyedidia/micro" do
       system "make", "build-quick"
       bin.install "micro"
```

Comment:
This runs automatically at the end of the install method. But it only runs in the buildpath. Since everything has been relocated to a subdirectory, we unfortunately need to intervene and run it manually in the subdirectory.

It installs the readme, license, etc.